src/main/resources/application-local.yml

```diff
@@ -3,6 +3,13 @@ server:
   servlet:
     context-path:
 
+spring:
+  main:
+    allow-bean-definition-overriding: true
+  servlet:
+    multipart:
+      max-file-size: 20MB
+      max-request-size: 21MB
```

Comment:
Bør max-file-size ligge i application? 🧐

Reply:
Kun brukt i application-local for mock